class-tgm-plugin-activation.php

```diff
@@ -1034,8 +1032,8 @@ public function force_deactivation() {
          */
         public static function get_instance() {
 
-            if ( ! isset( self::$instance ) && ! ( self::$instance instanceof TGM_Plugin_Activation ) ) {
-                self::$instance = new TGM_Plugin_Activation();
+            if ( ! isset( self::$instance ) && ! ( self::$instance instanceof self ) ) {
+                self::$instance = new self();

```

Comment:
Instead of having an untestable singleton class, how about we put a static into `load_tgm_plugin_activation()` instead?